### PR TITLE
1153 update the cloud recording release notes to incorporate new releases

### PR DESCRIPTION
**Pull request by atovpeko**

### CODE REVIEW

File: cloud-recording/reference/release-notes.md [hunk 1]
```diff
@@ -9,8 +9,44 @@ description: >
...
 
-Agora Cloud Recording is an add-on service to record and save voice calls, video calls, and interactive streaming on your cloud storage. Compared with Agora On-Premise Recording, Agora Cloud Recording is more efficient and convenient as it does not require deploying Linux servers.
+Agora Cloud Recording is an add-on service to record and save voice calls, video calls, and interactive streaming on your cloud storage. 
+Compared with Agora On-Premise Recording, Agora Cloud Recording is more efficient and convenient as it does not require deploying Linux servers.
 
+### Compatibility
+
+Agora Cloud Recording is compatible with the following SDKs:
+
+- Agora SDK for Android, iOS, macOS, and Windows v1.7.0 or later.
```
Comment (billy-the-fish) [inline cloud-recording/reference/release-notes.md:19]: Can you check the supported platforms and make sure this information is there and make a link to it please. Also check for broken links in supported platforms. (I just saw one. )

File: cloud-recording/reference/release-notes.md [hunk 2]
```diff
@@ -9,8 +9,44 @@ description: >
...
+**Mobile web page mode for web page recording**
+
+This release adds mobile web page mode for web page recording. The API changes are as follows:
+
+- Adds the `mobile` field in `extensionServiceConfig`. After setting `mobile` to `true`, the recording service uses the mobile web page mode to record the current web page.
+- The value range of the `videoWidth` and `videoHeight` fields in `extensionServiceConfig` is changed to [240, 1920].
+
+**Transcoder script for high availability mechanism**
+
+In composite recording mode, Agora provides a Transcoder script for merging the old M3U8 file with the new M3U8 file generated after the service enables the high availability mechanism into an MP4 file. For details, see [Manage Recorded Files](./develop/manage-files#when-a-server-is-disconnected-or-the-process-killed).
```
Comment (billy-the-fish) [inline cloud-recording/reference/release-notes.md:49]: Why Transcoder and not transcoder?

File: cloud-recording/reference/rest-api/start.md [hunk 3]
```diff
@@ -780,6 +784,49 @@ https://api.agora.io/v1/apps/<yourappid>/cloud_recording/resourceid/<resourceid>
     }
 }
 ```
+
+### Postpone transcoding 
+```
```
Comment (billy-the-fish) [inline cloud-recording/reference/rest-api/start.md:789]: You have code inside code, the JSON is not rendering and is incorrectly formatted. Can you update please.

**Review by billy-the-fish — CHANGES_REQUESTED** [review]
Nearly there.

File: cloud-recording/reference/release-notes.md (hunk 1 above)
Comment (atovpeko) [reply cloud-recording/reference/release-notes.md:19]: The [Supported Platforms](https://docs.agora.io/en/cloud-recording/reference/supported-platforms) page lists, to quote, "... platforms and products you use to develop apps that interact with Agora SD-RTN™". This, however, it about Agora products being compatible between themselves.

File: cloud-recording/reference/rest-api/start.md (hunk 3 above)
Comment (atovpeko) [reply cloud-recording/reference/rest-api/start.md:789]: Fixed

**Review by billy-the-fish — APPROVED** [review]
LGTM

**Pull request merged**